airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2036,6 +2036,19 @@ paths:
           $ref: "#/components/responses/NotFoundResponse"
         "422":
           $ref: "#/components/responses/InvalidInputResponse"
+  /v1/web_backend/geographies/list:
+    post:
+      tags:
+        - web_backend
+      summary: Returns two-letter country codes that can be selected for running data syncs in a particular geography.
```

Comment:
Should we also mention that we may return `auto`?

We can also add that users can expect the country codes to follow this iso standard: https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2 (minus `auto`), since that's what the frontend will be expecting so they can easily link them to country flags.